ang/civicase-base/providers/case-type-category.provider.js

```diff
@@ -82,6 +84,22 @@
       return caseTypeCategory.instance_id === instanceID;
     }
 
+    /**
+     * Get instance object for the sent case type category value
+     *
+     * @param {string} caseTypeCategoryValue case type category value
+     * @returns {boolean} if the sent case type category is part of the sent instance
+     */
+    function getInstance (caseTypeCategoryValue) {
```

Comment:
`getCaseTypeCategoryInstance` since there are multiple ways to find the instance, this particular one is using the case type category value.